### PR TITLE
Add ragtag v1.0.0

### PR DESCRIPTION
**Pull request by malonge**
I added a recipe for RagTag v1.0.0 which is already available on [PyPI](https://pypi.org/project/RagTag/1.0.0/)

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by malonge** [comment]
@BiocondaBot please add label

File: recipes/ragtag/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+test:
+  imports:
+    - ragtag_utilities
+
+about:
+  home: "https://github.com/malonge/RagTag"
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (npavlovikj) [inline recipes/ragtag/meta.yaml:38]: You can add the LICENSE file from the repo in the current recipe dir, and use `license_file: LICENSE` here.

File: recipes/ragtag/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  imports:
+    - ragtag_utilities
+
+about:
+  home: "https://github.com/malonge/RagTag"
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: "Fast reference-guided genome assembly scaffolding"
+  doc_url:
```
Comment (npavlovikj) [inline recipes/ragtag/meta.yaml:40]: `doc_url: https://github.com/malonge/RagTag/wiki/Usage`

File: recipes/ragtag/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - ragtag_utilities
+
+about:
+  home: "https://github.com/malonge/RagTag"
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: "Fast reference-guided genome assembly scaffolding"
+  doc_url: 
+  dev_url:
```
Comment (npavlovikj) [inline recipes/ragtag/meta.yaml:41]: `dev_url: https://github.com/malonge/RagTag`

File: recipes/ragtag/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  license: MIT
+  license_family: MIT
+  license_file: 
+  summary: "Fast reference-guided genome assembly scaffolding"
+  doc_url: 
+  dev_url: 
+
+extra:
+  recipe-maintainers:
+    - malonge
```
Comment (npavlovikj) [inline recipes/ragtag/meta.yaml:45]: You can also add the DOI info in the `extra` section above, e.g.:
```
extra:
  identifiers:
    - doi:10.1186/s13059-019-1829-6
```

File: recipes/ragtag/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  run:
+    - intervaltree
+    - numpy
+    - pysam
+    - python >3
+
+test:
+  imports:
+    - ragtag_utilities
+
```
Comment (npavlovikj) [inline recipes/ragtag/meta.yaml:33]: You should also test for the Python scripts here as well, e.g.:
```
test:
  commands:
    - ragtag.py --help
```

File: recipes/ragtag/meta.yaml (hunk 1 above)
Comment (malonge) [reply recipes/ragtag/meta.yaml]: fixed in the latest commit - thanks

File: recipes/ragtag/meta.yaml (hunk 2 above)
Comment (malonge) [reply recipes/ragtag/meta.yaml]: fixed in the latest commit - thanks

File: recipes/ragtag/meta.yaml (hunk 3 above)
Comment (malonge) [reply recipes/ragtag/meta.yaml]: fixed in the latest commit - thanks

File: recipes/ragtag/meta.yaml (hunk 4 above)
Comment (malonge) [reply recipes/ragtag/meta.yaml:47]: fixed in the latest commit - thanks

File: recipes/ragtag/meta.yaml (hunk 5 above)
Comment (malonge) [reply recipes/ragtag/meta.yaml:35]: fixed in the latest commit - thanks

**Comment by npavlovikj** [comment]
@BiocondaBot please fetch artifacts

**Comment by npavlovikj** [comment]
@BiocondaBot please fetch artifacts

**Comment by npavlovikj** [comment]
@BiocondaBot please merge

**Comment by npavlovikj** [comment]
Thank you for contributing @malonge 🎉

**Comment by malonge** [comment]
Thank you @npavlovikj !

**Pull request merged**